processor/deltatocumulativeprocessor/processor_test.go

```diff
@@ -282,3 +284,27 @@ func stream() SumBuilder {
 	_, base := sum.Stream()
 	return SumBuilder{Metric: sum, base: base}
 }
+
+func TestIgnore(t *testing.T) {
+	proc, sink := setup(t, nil)
+
+	dir := "./testdata/notemporality-ignored"
+	open := func(file string) pmetric.Metrics {
+		t.Helper()
+		md, err := golden.ReadMetrics(filepath.Join(dir, file))
+		require.NoError(t, err)
+		return md
+	}
+
+	in := open("in.yaml")
+	out := open("out.yaml")
+
+	ctx := context.Background()
+
+	err := proc.ConsumeMetrics(ctx, in)
+	require.NoError(t, err)
+
+	if diff := compare.Diff([]pmetric.Metrics{out}, sink.AllMetrics()); diff != "" {
+		t.Fatal(diff)
```

Comment:
t.Fatal? Why not "assert.Fail()", as the other ones? Or `assert.Len(t, "", compare.Diff(...))` ?